Mathlib/Algebra/Group/Submonoid/BigOperators.lean

```diff
@@ -23,7 +23,6 @@ submonoid, submonoids
 -/
 
 -- We don't need ordered structures to establish basic membership facts for submonoids
-assert_not_exists OrderedSemiring
```

Comment:
This `assert_not_exists` should probably be replaced by something else instead of merely dropped